integration_tests/e2e/dashboard.cy.ts

```diff
@@ -150,6 +151,25 @@ context('Searching for a report', () => {
     }
   })
 
+  it('should include location options to filter by', () => {
+    cy.clearCookies()
+    cy.resetBasicStubs({ user: mockDataWarden })
+    cy.signIn()
+
+    cy.task('stubIncidentReportingApiGetReports')
+    cy.visit('/reports')
+    const dashboardPage = Page.verifyOnPage(DashboardPage)
+    dashboardPage.locationOptions.then(locationOptions => {
+      expect(locationOptions).to.deep.equal([
+        { label: 'All locations', value: '' },
+        { label: 'All PECS regions', value: '.PECS' },
+        { label: 'Moorland (HMP & YOI)', value: 'MDI' },
+        { label: 'PECS North', value: 'NORTH' },
+        { label: 'PECS South', value: 'SOUTH' },
```

Comment:
Are we putting the specific PECS region at the very bottom - separating them from 'All PECS regions' - because we think filtering by specific PECS region is a relatively infrequent thing users will do?